UnrealPlugin/Source/PlayfabGSDK/Private/GSDKReadyForPlayersAction.cpp

```diff
@@ -24,7 +24,7 @@ UGSDKReadyForPlayersAction* UGSDKReadyForPlayersAction::ReadyForPlayers()
 
 void UGSDKReadyForPlayersAction::Activate()
 {
-	if (!UGSDKUtils::ReadyForPlayers())
+	if (!UGSDKUtils::MarkAsReadyForPlayers())
```

Comment:
This isnt enough, you need to attach yourself to the events from the gsdk and broadcast the preexisting ones. Alternatively the whole class could be deleted as well.